src/ucp/stream/stream_recv.c

```diff
@@ -268,38 +272,83 @@ UCS_PROFILE_FUNC(ucs_status_ptr_t, ucp_stream_recv_nb,
                  ucp_ep_h ep, void *buffer, size_t count,
                  ucp_datatype_t datatype, ucp_stream_recv_callback_t cb,
                  size_t *length, unsigned flags)
+{
+    ucp_request_param_t param = {
+        .op_attr_mask   = UCP_OP_ATTR_FIELD_DATATYPE |
+                          UCP_OP_ATTR_FIELD_CALLBACK |
+                          UCP_OP_ATTR_FIELD_FLAGS,
+        .cb.recv_stream = (ucp_stream_recv_nbx_callback_t)cb,
+        .flags          = flags,
+        .datatype       = datatype
+    };
```

Comment:
```
ucp_request_param_t param;

param.op_attr_mask   = ...
param.cb.recv_stream = ...
...
```
2 reasons:
 - pre-C99 compilers support
 - extra zero-ing of other fields on fast path

Reply:
i think we can keep as is for now since this is what we have in all other cases